lua/scriptwrapper.lua

```diff
@@ -15,6 +15,8 @@ package.path = EngineDir .. "/lua/?.lua"
 local sandbox = require('sandbox')
 local stp = require('stacktraceplus')
 
+local Sandboxed
```

Comment:
Seems to me this should be `sandboxed` (only function names appear to be Uppercase) or, even better in my opinion, `isSandboxed` (or something similar).